traits/adaptation/tests/benchmark.py

```diff
@@ -27,15 +31,15 @@ class IFoo{i}(Interface):
 class Foo{i}(HasTraits):
     pass
 """
-for i in range(N_SOURCES):
```

Comment:
I think it's fine to leave all these as `range`. It would be more efficient to be using `xrange` in Python 2, but it would also be a change from the previous behaviour, and we don't need to do it to make this code work under both Python 2 and Python 3.